apple/Sources/Sargon/Extensions/Methods/Profile/MFA/SecurityShieldBuilder+Wrap+Functions.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import SargonUniFFI
+
+extension SecurityShieldBuilder {
```

Comment:
👍

Reply:
Kept `sortedFactorSourcesForSelection` as a static function since it’s just a utility function and doesn’t need to involve the shield builder or imply that it can be modified.